ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <v-card flat rounded id="mhr-home-civic-address" class="mt-8 pa-8">
+    <v-row no-gutters class="pt-1">
+      <v-col cols="12" sm="2">
+        <label class="generic-label" :class="{'error-text': false}">Civic Address</label>
+      </v-col>
+      <v-col cols="12" sm="10" class="mt-n1">
+
+      <div class="base-address">
+        <!-- Display fields -->
+        <!-- Edit fields -->
+
+          <v-form ref="addressForm" name="address-form" lazy-validation>
```

Comment:
There is an extra indent here. 

Also are these comments for development purposes?